lib/interceptor.js

```diff
@@ -31,8 +31,13 @@ module.exports = class Interceptor {
   constructor(scope, uri, method, requestBody, interceptorOptions) {
     const uriIsStr = typeof uri === 'string'
     // Check for leading slash. Uri can be either a string or a regexp, but
-    // we only need to check strings.
-    if (uriIsStr && /^[^/*]/.test(uri)) {
+    // we only need to check strings./tests/test_allow_unmocked.js
+
+    if (
+      uriIsStr &&
+      /^[^/*]|^\s*$/.test(uri) &&
```

Comment:
What's the second half of this regex doing?

Regexes are hard to read; I wonder if we could write the leading slash condition more clearly using `.startsWith()`.

Reply:
Initially I was going to use startsWith('/') but thought adding to the original regex which already checks for "/" || '*' would be a cleaner approach. The second half of the regex checks for an empty string or a string of whitespaces. I would be happy to change to startsWith('/'). Would you like continued support for ('*') as the current code does or require ('/*')?